src/script/stack.zig

```diff
@@ -158,6 +158,25 @@ pub const Stack = struct {
         return std.mem.readVarInt(i64, elem, native_endian);
     }
 
+    /// Removes an item from the stack at the specified index
+    ///
+    /// # Arguments
+    /// - `index`: The index of the item to remove (0 is the top of the stack)
+    ///
+    /// # Returns
+    /// - `StackError` if the index is out of bounds
+    /// - `[]u8`: The removed item
+    pub fn removeAt(self: *Stack, index: usize) StackError![]u8 {
```

Comment:
i don't think this method should be public. It's only a tool to help writing the opcodes. It shouldn't be called from outside as it is not an opcode

Reply:
I moved the functionality directly into the opcode opRoll since this function is basically just orderedRemove anyways.